python_modules/dagster/dagster/_core/definitions/declarative_scheduling/asset_condition.py

```diff
@@ -199,7 +200,7 @@ class AssetConditionEvaluationState:
     previous_tick_evaluation_timestamp: Optional[float]
 
     max_storage_id: Optional[int]
-    extra_state_by_unique_id: Mapping[str, PackableValue]
+    extra_state_by_unique_id: Mapping[str, Any]
```

Comment:
hmmm this seems like a regression. What is going on here?

Reply:
Generally like to leave comments in code saying intent is to type it later just in case a stack gets caught in limbo for awhile